lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2946,9 +2947,13 @@ ParseResult FIRCircuitParser::parseModule(CircuitOp circuit,
 
         // If we got to the next module, then we're done.
       case FIRToken::kw_module:
-      case FIRToken::kw_extmodule:
-        return success();
-
+      case FIRToken::kw_extmodule: {
+        // If the module is not indented to at most 2 it's actually an
+        // identifier
+        auto indent = getIndentation();
+        if (indent.has_value() && indent <= moduleIndent)
```

Comment:
As @dtzSiFive pointed out, hardcoding the width amount here goes against the spec. I think you can just grab `getIdentation` before this `while` loop and compare the `getIndentation` here to that.